### PR TITLE
Query passengers and contact details

### PR DESCRIPTION
**Pull request by KarinaDavtyan**
Summary: Querying passengers and contact details from `AllBookings.json`

Note: `visaRequired` field is **hardcoded** and set to `false` for now

### CODE REVIEW

File: apps/graphql/src/dataloaders/bookingsLoader/BookingsLoader.js [hunk 1]
```diff
@@ -18,17 +19,38 @@ const sanitizeBookings = (
...
+      },
+      passengers,
       type,
       ...typeSpecificData,
     };
   });
 };
 
+const sanitizePassengers = (booking: BookingApiResult): Array<Passenger> =>
+  booking.passengers.map((passenger, index) => {
```
Comment (tbergq) [inline apps/graphql/src/dataloaders/bookingsLoader/BookingsLoader.js]: Index is not used, can you please remove it?

We should update our eslint to catch these ones in the future.

File: apps/graphql/src/types/output/Passenger.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      type: GraphQLString,
+    },
+    category: {
+      type: GraphQLString,
+    },
+    firstname: {
+      type: GraphQLString,
+    },
+    id: {
+      type: GraphQLInt,
```
Comment (tbergq) [inline apps/graphql/src/types/output/Passenger.js]: Can you make this a global id instead?

File: apps/graphql/src/types/output/Passenger.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    },
+    insurance_type: {
```
Comment (tbergq) [inline apps/graphql/src/types/output/Passenger.js]: please use camelCase

File: apps/graphql/src/types/output/Passenger.js (hunk 2 above)
Comment (tbergq) [reply apps/graphql/src/types/output/Passenger.js]: https://github.com/kiwicom/margarita/blob/master/apps/graphql/src/types/output/BookingInterface.js#L16

File: apps/graphql/src/types/output/Passenger.js (hunk 3 above)
Comment (KarinaDavtyan) [reply apps/graphql/src/types/output/Passenger.js]: Can I add camelcase eslint rule, as snake case keep happening again and again
If yes, can I put `ItinerariesMock.js` in `.eslintignore` or I could just disable this rule for this file

Comment (tbergq) [reply apps/graphql/src/types/output/Passenger.js]: > Can I add camelcase eslint rule

Please do not do it for now, as we get most the rules from `@kiwicom/eslint-config` I will have a look at it and see if we can improve it for our needs.

Comment (KarinaDavtyan) [reply apps/graphql/src/types/output/Passenger.js]: Ok sure, wont touch it! I ll just fix it to `insuranceType` then

File: apps/graphql/src/dataloaders/bookingsLoader/BookingsLoader.js (hunk 1 above)
Comment (KarinaDavtyan) [reply apps/graphql/src/dataloaders/bookingsLoader/BookingsLoader.js]: yes, missed that. Is removed now

File: apps/graphql/src/types/output/Passenger.js (hunk 2 above)
Comment (KarinaDavtyan) [reply apps/graphql/src/types/output/Passenger.js]: Ok, changed to global id now

File: apps/graphql/src/types/output/Passenger.js [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+  GraphQLInt,
+  GraphQLBoolean,
+} from 'graphql';
+import globalID from '@kiwicom/graphql-global-id';
+
+export default new GraphQLObjectType({
+  name: 'Passenger',
+  description: 'Passenger details',
+  fields: {
+    id: globalID(({ bid }) => bid),
```
Comment (tbergq) [inline apps/graphql/src/types/output/Passenger.js]: `id: globalID(({ id }) => id)`?

Comment (KarinaDavtyan) [reply apps/graphql/src/types/output/Passenger.js]: 😓 changing now

**Pull request merged**